template.yaml

```diff
@@ -34,12 +38,15 @@ Parameters:
     Type: String
     Description: Minimum capacity for provisioned concurrency
     Default: 1
-  EventBusName:
-    Type: AWS::SSM::Parameter::Value<String>
-    Default: nvaEventBusName
-  EventBusArn:
-    Type: AWS::SSM::Parameter::Value<String>
-    Default: nvaEventBusArn
+  Suffix:
```

Comment:
follow the pattern of the nva-publicaiton-api and make the template closer to ready for the CI/CD pipeline